.circleci/config.yml

```diff
@@ -106,36 +69,68 @@ jobs:
 
 
 workflows:
-  version: 2
-  build-test-deploy:
+  ci:
     jobs:
-      - build:
+      - docker/publish:
+          name: Docker build test
+          before_build: &version
+            - run:
+                name: Generate build version.json
+                command: >
+                  printf 
+                  '{"commit":"%s","version":"%s", "source":"https://github.com/%s/%s","build":"%s"}\n' 
```

Comment:
whitespace before "source". Not sure if it matters but is inconsistent with the old

Reply:
fixed